storage/storage_image.go

```diff
@@ -1207,7 +1207,7 @@ func (s *storageImageDestination) Commit(ctx context.Context, unparsedToplevel t
 		if len(oldNames) > 0 {
 			names = append(names, oldNames...)
```

Comment:
… Then the `names` variable will have a trivial value and can be eliminated/inlined.